app/views/RoomsListView/index.tsx

```diff
@@ -154,7 +154,8 @@ const getItemLayout = (data: ISubscription[] | null | undefined, index: number,
 	offset: height * index,
 	index
 });
-const keyExtractor = (item: ISubscription) => item.rid;
+// We need to pass the isSearching to handle the room if it's a room returned from search (without observable) or from query subscription (with observable)
```

Comment:
This is wrong. The PR is correctly fixing the bug, but because of a different reason :)

When we load FlatList, `chats` contains the list of rooms.
When we go to search, `search` starts empty and we first render the list empty. When `search` has data, we show them.
When we go back to `chats`, we clean `search`, but `chats` already has the previous content, so it reuses the same cells with the same key from `keyExtractor`.
There's no reason to unmount the cell, which doesn't trigger `useEffect` again, hence there's no observe.